src/js/language_label.js

```diff
@@ -40,77 +40,309 @@ let languageNames = new Intl.DisplayNames(Label.getLocales(), {
   type: "language",
 });
 let langCodes = [
-  "am",
-  "ar",
-  "az",
-  "be",
-  "bg",
-  "br",
-  "bs",
-  "ca",
-  "co",
-  "cs",
-  "cy",
-  "da",
-  "de",
-  "el",
-  "en",
-  "eo",
-  "es",
-  "et",
-  "eu",
-  "fi",
-  "fr",
-  "fy",
-  "ga",
-  "gd",
-  "he",
-  "hi",
-  "hr",
-  "hu",
-  "hy",
-  "id",
-  "is",
-  "it",
-  "ja",
-  // "ja_kana",
-  "ja-Latn",
-  "ja-Hira",
-  "ka",
-  "kk",
-  "kn",
-  "ko",
-  "ko-Latn",
-  "ku",
-  "la",
-  "lb",
-  "lt",
-  "lv",
-  "mk",
-  "mt",
-  "ml",
-  "nl",
-  "no",
-  "oc",
-  "pl",
-  "pt",
-  "rm",
-  "ro",
-  "ru",
-  "sk",
-  "sl",
-  "sq",
-  "sr",
-  "sr-Latn",
-  "sv",
-  "ta",
-  "te",
-  "th",
-  "tr",
-  "uk",
-  "zh",
-].map((id) => {
-  return { id, name: languageNames.of(id) };
+  { id: "am" },
+  { id: "ar" },
+  { id: "az" },
+  { id: "be" },
+  { id: "bg" },
+  { id: "br" },
+  { id: "bs" },
+  { id: "ca" },
+  { id: "co" },
+  { id: "cs" },
+  { id: "cy" },
+  { id: "da" },
+  { id: "de" },
+  { id: "el" },
+  { id: "en" },
+  { id: "eo" },
+  { id: "es" },
+  { id: "et" },
+  { id: "eu" },
+  { id: "fi" },
+  { id: "fr" },
+  { id: "fy" },
+  { id: "ga" },
+  { id: "gd" },
+  { id: "he" },
+  { id: "hi" },
+  { id: "hr" },
+  { id: "hu" },
+  { id: "hy" },
+  { id: "id" },
+  { id: "is" },
+  { id: "it" },
+  { id: "ja" },
+  // { id: "ja_kana"},
+  { id: "ja-Latn" },
+  { id: "ja-Hira" },
+  { id: "ka" },
+  { id: "kk" },
+  { id: "kn" },
+  { id: "ko" },
+  { id: "ko-Latn" },
+  { id: "ku" },
+  { id: "la" },
+  { id: "lb" },
+  { id: "lt" },
+  { id: "lv" },
+  { id: "mk" },
+  { id: "ml" },
+  { id: "mt" },
+  { id: "nl" },
+  { id: "no" },
+  { id: "oc" },
+  { id: "pl" },
+  { id: "pt" },
+  { id: "rm" },
+  { id: "ro" },
+  { id: "ru" },
+  { id: "sk" },
+  { id: "sl" },
+  { id: "sq" },
+  { id: "sr" },
+  { id: "sr-Latn" },
+  { id: "sv" },
+  { id: "ta" },
+  { id: "te" },
+  { id: "th" },
+  { id: "tr" },
+  { id: "uk" },
+  { id: "zh" },
+
+  // Indigenous languages of the United States
+
+  // These languages are grouped by linguistic family, in loose geographical
+  // order from northwest to southeast. The English name of each language is
+  // included as a fallback, in case CLDR is missing data on language names.
```

Comment:
I wonder if sorting the list alphabetically by language code would be easier to maintain than this geographical sort order. We could easily end up with duplicates.